promotion/controllers.go

```diff
@@ -28,12 +28,15 @@ import (
 // Router for promotion endpoints
 func Router(service *Service) chi.Router {
 	r := chi.NewRouter()
+	createPromotion := CreatePromotion(service)
+	createClaims := CreateClaims(service)
 	if os.Getenv("ENV") != "local" {
```

Comment:
maybe we should check more concretely for `ENV = development` or `ENV = staging`? - I don't think we will use this on production initially

Reply:
moved check into handler [87b0d3c](https://github.com/brave-intl/bat-go/pull/338/commits/87b0d3cb7cdad85e68470bd1195984eae23b96db)